Sources/UI/DataSourceDelegate/Swift/Table View/Animator/TableCellAnimator.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  TableCellAnimator.swift
+//  Catalog
+//
+//  Created by Zvonimir Medak on 15.09.2022..
+//  Copyright © 2022 Infinum. All rights reserved.
+//
+// Inspiration from: https://www.vadimbulavin.com/tableviewcell-display-animation/
+
+import UIKit
+
+public final class TableCellAnimator {
```

Comment:
I would use this one as a base class or a protocol, and make second one ⬇️ (slide left right) one of the concrete implementations of it 😄

Reply:
We could make animator a base class so that we get to extend that for different types of behavior over animations. However, why do you feel like we'd be better off with different animators instead of different animation types?

Under the assumption that the base animator uses the same logic to execute a particular animation, this makes sense to me. We can do both, though, that way we get to support different types of animations for different animators. 😄